tests/baseline_stop_on_first_error.txt

```diff
@@ -55,3 +55,4 @@ bom.ini: e=0 user=107
 ... single1=def;
 ... single2=qrs;
 duplicate_sections.ini: e=0 user=108
+no_value.ini: e=2 user=108
```

Comment:
Weird -- I couldn't see immediately why user was not incremented here. Is this a regression?

Reply:
Debugged this one. The flow is as follows - it reads the '[section_list]', then it jumps out of `ini_parse` because because the second line doesn't contain a value. The handler is never called. It was in general the behavior as before - the whole section would actually be ignored, if it contained just keys and no values.

In other cases `INI_STOP_ON_FIRST_ERROR` is zero by default, so `ini_parse` goes ahead and finds other sections, so the handler is called. This doesn't seem a regression.

Thanks.